sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -154,6 +193,43 @@ def __init__(self, locations: List[sbol3.Location] = [], **kwargs) -> None:
             self.qualifier_value    = sbol3.TextProperty(self, f"{self.GENBANK_FEATURE_QUALIFIER_NS}#value", 0, math.inf)
 
 
+    # class Range_GenBank_Extension(sbol3.Range):
+    #     """Overrides the sbol3 Range class to include fields to store the  
+    #     start and end position types (AfterPostion / BeforePosition / ExactPosition).
+    #     :extends: sbol3.Range class
+    #     """
+    #     GENBANK_RANGE_NS = "http://www.ncbi.nlm.nih.gov/genbank#locationPosition"
+    #     def __init__(self, sequence: sbol3.Sequence = sbol3.Sequence("autoCreatedSequence"), start: int = 0, end: int = 0, **kwargs) -> None:
+    #         # instantiating sbol3 SequenceFeature object
+    #         super().__init__(sequence = sequence, start = start, end = end, **kwargs)
+    #         # Setting properties for GenBank's location position not settable in any SBOL3 field.
+    #         self.start_position = sbol3.IntProperty(self, f"{self.GENBANK_RANGE_NS}#start", 0, 1)
+    #         self.end_position   = sbol3.IntProperty(self, f"{self.GENBANK_RANGE_NS}#end"  , 0, 1)
+
+
+    class Location_GenBank_Extension(sbol3.Location):
+        """Overrides the sbol3 Location class to include fields to store the  
+        start and end position types (AfterPostion / BeforePosition / ExactPosition).
+        :extends: sbol3.Location class
+        """
+        # Use the SBOL3 namespace for the type URI as a workaround for
+        # a bug in pySBOL3.
+        # TODO: use genbank namespace when the pySBOL3 bug is fixed
+        # NOTE: pySBOL3 BUG REPORT: https://github.com/SynBioDex/pySBOL3/issues/414
+        # NOTE: pySBOL3 BUG FIX   : https://github.com/SynBioDex/pySBOL3/pull/415
```

Comment:
- [ ] Please add an issue for removing the workaround after the next pySBOL3 release.

Reply:
Done, created at https://github.com/SynBioDex/SBOL-utilities/issues/187